src/h.js

```diff
@@ -1,6 +1,8 @@
 import { VNode } from './vnode';
 import options from './options';
+import { iterableToArray, isFunction } from './util';
 
+const hasSymbolSupport = isFunction(Symbol) && Symbol.iterator;
```

Comment:
`isFunction(Symbol)` triggers a `ReferenceError` if Symbol is undefined. Using `typeof Symbol === 'function'` is OK however.